<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pre_bill_generation</class>
 <widget class="QWidget" name="pre_bill_generation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QComboBox" name="month">
       <item>
        <property name="text">
         <string>Baisakh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jestha</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ashad</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shrawan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bhadra</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ashoj</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kartik</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mangsir</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poush</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Margh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Falgun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chaitra</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Class</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="class_2"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>print</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Pre-bill-Generation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Month</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Class</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Particulars</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Previous balance</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
